var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -75,6 +76,7 @@ class NetcdfC(AutotoolsPackage):
     variant("dap", default=False, description="Enable DAP support")
     variant("jna", default=False, description="Enable JNA support")
     variant("fsync", default=False, description="Enable fsync support")
+    variant("zstd", default=False, description="Enable ZStandard compression")
```

Comment:
```suggestion
    variant("zstd", default=False, description="Enable ZStandard compression", when="@4.9.0:")
```

Reply:
Nice, I didn't think variant could do `when=`.

Reply:
The corresponding conflict should be removed, I guess.